Source/SimpleActions/Public/Structs/SimpleActionSingle.h

```diff
@@ -20,8 +20,13 @@ struct SIMPLEACTIONS_API FSimpleActionSingle
 	
 public:
 
-	UPROPERTY(EditAnywhere, BlueprintReadOnly, Instanced, meta = (ShowOnlyInnerProperties))
```

Comment:
IIRC this was necessary to prevent the actions' properties from being nested in groups.